src/pokemon_summary_screen.c

```diff
@@ -2303,16 +2303,26 @@ static void BufferMonMoveI(u8 i)
 
     if (gBattleMoves[sMonSummaryScreen->moveIds[i]].power <= 1)
         StringCopy(sMonSummaryScreen->summary.movePowerStrBufs[i], gText_ThreeHyphens);
+    else if (gBattleMoves[sMonSummaryScreen->moveIds[i]].accuracy == 235)
+        StringCopy(sMonSummaryScreen->summary.movePowerStrBufs[i], gText_360); // Once again this is the HACKIEST of hacks, but hey.
+    
+    else if (gBattleMoves[sMonSummaryScreen->moveIds[i]].accuracy == 234)
+        StringCopy(sMonSummaryScreen->summary.movePowerStrBufs[i], gText_270);
+        //You didn't work on this project for reasonable coding decisions.
+        //See below for explanation and justification
+    
     else
         ConvertIntToDecimalStringN(sMonSummaryScreen->summary.movePowerStrBufs[i], gBattleMoves[sMonSummaryScreen->moveIds[i]].power, STR_CONV_MODE_RIGHT_ALIGN, 3);
 
     if (gBattleMoves[sMonSummaryScreen->moveIds[i]].accuracy == 0)
         StringCopy(sMonSummaryScreen->summary.moveAccuracyStrBufs[i], gText_ThreeHyphens);
     else if (gBattleMoves[sMonSummaryScreen->moveIds[i]].accuracy == 238) // Ideally this would be gBattleMoves[sMonSummaryScreen->moveIds[i]].SOMETHING == MOVE_STUN_SPORE, but idk how to get that.
```

Comment:
I would imagine simply `sMonSummaryScreen->moveIds[i] == MOVE_STUN_SPORE` would do the trick, but if not, this is a solution for now lol.